src/Microsoft.ServiceFabric.AspNetCore/AspNetCoreCommunicationListener.cs

```diff
@@ -2,19 +2,19 @@
 // Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
+using System;
```

Comment:
Could you please move these usings inside the namespace below, this is the convention followed in Service fabric repositories in external as well as internal.